packages/parcel-transformer-autometrics/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import { Transformer } from "@parcel/plugin";
+import { addAutometricsOptions } from "./lib";
+
+export default new Transformer({
+  async transform({ asset }) {
+    const { filePath } = asset;
+    const code = await asset.getCode();
+
+    if (/\.(js|ts)x?$/.test(filePath) && code.includes("autometrics(")) {
```

Comment:
Depending on the formatting, there could be whitespace between `autometrics` and `(`. Might be better to use a regular expression or just check whether it includes `autometrics` (false positives don't have much harm except a little wasted performance, am I right?).

Reply:
Hmmm, that's a good point, I didn't think about that. 

Yeah, false positives basically waste the build performance, nothing else. I'll remove the `(` and we can look into improving the performance if necessary